src/shogun/lib/type_case.h

```diff
@@ -145,7 +145,7 @@ namespace shogun
 #undef SG_ADD_SGVECTOR_TYPE
 #undef SG_ADD_SGMATRIX_TYPE
 
-		std::string print_map(const typemap& map)
+		static inline std::string print_map(const typemap& map)
```

Comment:
if you want to inline things use `SG_FORCE_INLINE`

Reply:
this is an inline specifier for the linker, the code itself doesn't have to be inlined, so don't have to force it

Reply:
I see thanks!

Reply:
`SG_FORCED_INLINE` plz

Reply:
@vigsterkr this is to satisfy the one definition rule for the linker. should it be `SG_FORCED_INLINE`ed?

Reply:
yes plz force it